examples/grpc-unary-non-blocking/grpc_unary_non_blocking.out

```diff
@@ -1,13 +1,13 @@
 # Create a Ballerina project and a module inside it.
 # Generate mock service and stub code using `grpc_unary_non_blocking_cont.proto` file.
 # For example, if you create a module named 'service', execute the following command to generate Ballerina code.
-$ ballerina grpc --input grpc_unary_non_blocking_cont.proto --output service --mode service
+$ ballerina grpc --input grpc_unary_non_blocking_cont.proto --output src/service --mode service
 
-# Once you run the command, `grpc_unary_non_blocking_cont_pb.bal` and `HelloWorld_sample_service.bal` files are generated inside service directory.
+# Once you run the command, `grpc_unary_non_blocking_cont_pb.bal` and `HelloWorld_sample_service.bal` files are generated inside service module.
 # Replace the content of `HelloWorld_sample_service.bal` file with the content of `grpc_unary_non_blocking.bal` file.
 
 # Execute the following command to build the 'service' module.
 $ ballerina build service
 
-# Run the client using the following command.
+# Run the service using the following command.
 $ ballerina run target/service.balx
```

Comment:
Need a new line here.

Reply:
Fixed in 6f7382cc88d67c386cfcc5e2d50f8c481c55aa06